infra/modules/notifications-app/resources/variables.tf

```diff
@@ -0,0 +1,25 @@
+variable "name" {
+  type        = string
+  description = "Name of the notifications project/application"
+}
+
+variable "sender_email" {
+  type        = string
+  description = "Email address to use to send notification emails"
+}
```

Comment:
couldn't we get the sender_email by using a data source with the email_identity_arn and doing data.email_identity_arn.sender.email? that would simplify the variables a bit

Reply:
I mean, we can. But I think that would cause confusion when looking at the module inputs. You would see `sender_display_name` but not `sender_email`, and possibly wonder why you are inputting a display name if you aren't also inputting an email.

Reply:
I think it could make sense if we made display name more generically named so it looks like this:

```hcl
module "notifications" {
  count  = module.app_config.enable_notifications ? 1 : 0
  source = "../../modules/notifications/resources"

  name                = local.notifications_app_name
  domain_identity_arn = local.domain_identity_arn
  display_name        = local.notifications_config.display_name
}
```

Then you would use display name in other places... I think.

Reply:
I wonder if we should make display name a required attribute... it would make this line much simpler

```hcl
  from_address   = var.sender_email != null ? (var.display_name != null ? "${var.display_name} <${var.sender_email}>" : var.sender_email) : null
```

Reply:
Hmm, I actually don't see this

> a data source with the email_identity_arn and doing data.email_identity_arn.sender.email

here?

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/sesv2_email_identity

Reply:
Ah wait, I actually don't see this

> a data source with the email_identity_arn and doing data.email_identity_arn.sender.email

here?

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/sesv2_email_identity

Unless there's some terraform knowledge that I'm missing